pom.xml

```diff
@@ -76,6 +76,16 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
```

Comment:
From my understending Kafdrop uses `spring-kafka`. So I think that to update Kafka clients we have to update `spring-kafka`.
Is required for this PR to update kafka-clients? If not I suggest to remove this update.